relayer/app.go

```diff
@@ -578,13 +597,15 @@ func (a *App) Start(ctx context.Context, tunnelIDs []uint64) error {
 	scheduler := NewScheduler(
 		a.Log,
 		tunnelRelayers,
+		len(tunnels),
 		a.Config.Global.CheckingPacketInterval,
 		a.Config.Global.SyncTunnelsInterval,
 		a.Config.Global.MaxCheckingPacketPenaltyDuration,
 		a.Config.Global.PenaltyExponentialFactor,
 		isSyncTunnelsAllowed,
 		a.BandClient,
 		a.TargetChains,
+		chainNames,
 	)
 
```

Comment:
Can we move tunnel relayer logic in? so that we can use `sync tunnel` function to do the first one as well? instead of duplicate logic and sending them in?